pkg/features/list.go

```diff
@@ -87,6 +87,9 @@ var (
 	// Will aggregate to one edge per unique port/protocol/direction instead of one edge per unique IP/port/protocol/direction.
 	NetworkGraphAggregateExternalIPs = registerFeature("Aggregate all external IP graph edges, showing only unique port/protocol pairs", "ROX_NETWORK_GRAPH_AGGREGATE_EXT_IPS")
 
+	// NetworkFlowBatching enables batching of network flow updates to smooth out data spikes
+	NetworkFlowBatching = registerFeature("Enable batching of network flow updates to smooth out data spikes", "ROX_NETFLOW_BATCHING")
```

Comment:
I would like to also see the FF for limiting the cache size that would be separate FF from this one. The cache size is currently unlimited and it would be good to be able to enable batching without enabling the limit (or the other way round). That would give us more control over the feature. Would that be possible to add two fine-granular feature flags?